tests/acceptance/bootstrap/AuthContext.php

```diff
@@ -216,6 +216,29 @@ public function userRequestsEndpointsWithNoAuthentication(string $method, TableN
 		}
 	}
 
+	/**
+	 * @When a user sends :method request to the following url with no authentication
+	 *
+	 * @param $method
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function aUserSendsRequestToTheFollowingUrlWithNoAuthentication($method, TableNode $table): void {
```

Comment:
With addition of this new function we can revert changes in the function `userRequestsEndpointsWithNoAuthentication` made in PR https://github.com/owncloud/ocis/pull/10351.
others LGTM :+1:

Reply:
what is the difference?

Reply:
old ones takes endpoint without service column and new ones takes full-url with service column